src/js/models/v2/account/FederalAccount.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * FederalAccount.js - v2
```

Comment:
This type of model is in the v1 style  - please move to the top level `models` directory, or convert to a v2 model `BaseFederalAccount` object using JS prototypal inheritance.